core/src/main/java/org/p2p/core/token/TokenExtensions.kt

```diff
@@ -27,3 +28,7 @@ data class TokenExtensions(
 fun List<Token.Active>.filterTokensForWalletScreen(): List<Token.Active> {
     return filter { it.tokenExtensions.isTokenVisibleOnWalletScreen == true }
 }
+
+fun List<Token.Active>.filterIsUSDCToken(): List<Token.Active> {
+    return filter { it.tokenSymbol.equals(Constants.USDC_SYMBOL, ignoreCase = true) }
```

Comment:
i think there can be only one usdc token
also search usdc by mint address, not usdc

Reply:
I think it's okay to leave `filter` in the name, but it needs to be a `filterIsUsdcTokens` (tokenS instead of token), it returns a list of tokens